src/EditorFeatures/Core/Extensibility/NavigationBar/INavigationBarItemService.cs

```diff
@@ -10,19 +11,40 @@
 
 namespace Microsoft.CodeAnalysis.Editor
 {
+    [Obsolete("Legacy API for TypeScript.  Once TypeScript moves to IVSTypeScriptNavigationBarItemService", error: false)]
```

Comment:
Same comment as the other PR, I think it would be helpful if the comment here was expanded. But if TS plans to adopt these changes quickly then maybe it doesn't matter too much.